.node-version

```diff
@@ -1 +1 @@
-10.14.1
+10.18.0
```

Comment:
why not 12.x?

Reply:
I simply sticked to the minimum version to make it work. If you wish I can upgrade

Reply:
@gunzip we will implement an azure function for the backend, what's the best node version?

Reply:
any 12.x version using v3 runtime (10.x is EOL next year): https://nodejs.org/en/about/releases/